lib/util/format.ts

```diff
@@ -0,0 +1,14 @@
+import { getAddress } from 'viem';
+
+export function formatAddress(address: string, chars = 8) {
```

Comment:
maybe could call like shortAddressDisplay or something. This reads like it could be a checksum?

Reply:
good shout, will update